packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -176,16 +193,32 @@ const InputContainer = createComponentSlot("div")<FieldContainerClassKey, OwnerS
     `,
 );
 
+const getCommonHelperTextStyles = (theme: Theme) => css`
+    &.${formHelperTextClasses.root} {
+        display: flex;
+        gap: ${theme.spacing(1)};
+    }
```

Comment:
This is applied to each instance of `FormHelperText` directly, right? 
Is the additional selector really needed then? 🤔 

```suggestion
    display: flex;
    gap: ${theme.spacing(1)};
```